css/main.css

```diff
@@ -413,8 +419,10 @@ h1{
 	margin-left: 0.5em; /* To put space between scroll bar of main panel and this bar to avoid accidental scrolling. */
 }
 .mainPanel{
+	margin: 0 0 0 0.5em; /* Main panel only has a margin on the left so that iframe scroll bars aren't displaced. */
+	padding: 0.5em 0 0.5em 0; /* Content padding so we don't get that background color clipping over the top/bottom of content */
```

Comment:
another CSS shorthand for padding / margin, is that if you only supply two values, the first value is used for top and bottom, and then the second is used for left and right.
so we could do:
`padding: 0.5em 0;`

and it would still do the same thing.

https://developer.mozilla.org/en-US/docs/Web/CSS/padding#syntax

Reply:
Good call, I knew about that aspect of how it works but I just forgot to use it here, thanks for point it out.